imports/ui/TaskListItem.tsx

```diff
@@ -70,6 +112,29 @@ export function TaskListItem({
           task.plannedDate != null ? format(task.plannedDate, "p") : undefined
         }
       />
+      <ListItemSecondaryAction className={classes.listItemSecondaryAction}>
+        <IconButton edge="end" aria-label="postpone" onClick={postponeClicked}
+          hidden={task.plannedDate === undefined}>
+          <UpdateIcon />
+        </IconButton>
+        <Menu
+          id="postpone-menu"
+          anchorEl={anchorEl}
+          keepMounted
+          open={Boolean(anchorEl)}
+          onClose={closePostponeMenu}
+        >
+          <MenuItem onClick={postponeBy(1)}>+1h</MenuItem>
+          <MenuItem onClick={postponeBy(24)}>+1d</MenuItem>
+          <MenuItem onClick={postponeBy(24 * 7)}>+1w</MenuItem>
+        </Menu>
+        <div className={classes.listItemSecondaryActionSpacer} />
```

Comment:
Can you put another one of these after the trash icon?

Reply:
Done.